### PR TITLE
Updating Data Sources pages

### PR DESCRIPTION
**Pull request by chejennifer**

### CODE REVIEW

File: datasets/index.md [hunk 1]
```diff
@@ -20,23 +20,4 @@ Data in the Data Commons Graph comes from a variety of sources, each of which of
...
-This chart goes a level deeper and illustrates the total number of statistical variables available at the state level for each country worldwide.
-<p> &nbsp; </p>
-<div class="flourish-embed flourish-map" data-src="visualisation/17716780"></div>
-<p> &nbsp; </p>
-
-Finally, this third chart illustrates the total number of statistical variables available at the district/county level worldwide.
-
-<p> &nbsp; </p> 
-<div class="flourish-embed flourish-map" data-src="visualisation/17719383"></div>
-<p> &nbsp; </p>
```
Comment (juliawu) [inline datasets/index.md:42]: Is this deletion intentional? I think having these visualizations may have been an ask from James. @miss-o-soup

Comment (chejennifer) [reply datasets/index.md:42]: good catch, I just ran the script & I don't think this should be deleted. Let me update

Comment (chejennifer) [reply datasets/index.md]: done

**Review by juliawu — APPROVED** [review]
Thanks for updating!

**Pull request merged**